Sources/_URLRouting/URLRequestData.swift

```diff
@@ -68,17 +68,17 @@ public struct URLRequestData: Equatable, _EmptyInitializable {
     password: String? = nil,
     host: String? = nil,
     port: Int? = nil,
-    path: ArraySlice<Substring> = [],
+    path: String = "",
```

Comment:
Updated the `URLRequestData` "designated" initializer to take non-sub-sequences.